jenkinsfiles/ginkgo.Jenkinsfile

```diff
@@ -125,13 +125,13 @@ pipeline {
                         }
                     }
                 }
-                stage('Boot vms K8s-1.13 net-next kubeproxy-free') {
```

Comment:
This file also doesn't get run on master. @nbusseneau also a candidate for deletion?

Reply:
It is still ran from obsolete job https://jenkins.cilium.io/view/Obsolete/job/cilium-ginkgo/job/cilium/job/master/, but nobody looks at that it seems.

Reply:
Other than that I think it can safely be removed on `master`, yes.